hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ResourceAttributeEnricher.java

```diff
@@ -77,4 +86,15 @@ private boolean isValidEvent(Event event) {
         && (event.getAttributes() != null)
         && (event.getAttributes().getAttributeMap() != null);
   }
+
+  private String getDeploymentType(String hostName) {
+    for (Deployment d : Deployment.values()) {
+      if (d == Deployment.UNRECOGNIZED) {
+        break;
+      } else if (hostName.contains(EnrichedSpanConstants.getValue(d))) {
+        return EnrichedSpanConstants.getValue(d);
+      }
+    }
+    return EnrichedSpanConstants.getValue(Deployment.DEPLOYMENT_WEB);
```

Comment:
Do we really need to enrich the span if the pod name doesn't match any deployment type? A bit skeptical about giving the default deployment type as web for no match.

Reply:
That was part of the requirement to default deployment type to web. @Sunn-y-Arora thoughts..?